hoot-core/src/main/cpp/hoot/core/criterion/PowerLineCriterion.h

```diff
@@ -22,44 +22,34 @@
  * This will properly maintain the copyright information. DigitalGlobe
```

Comment:
AttributeDistanceExtractor was the same as AttributeScoreExtractor in logic so removed it

Reply:
...didn't mean for it to be converted to PowerLineCriterion.h, but oh well.